### PR TITLE
Warning about the host create help output

### PR DESCRIPTION
**Pull request by tstrachota**
Created based on PR https://github.com/theforeman/hammer-cli-foreman/pull/181

### CODE REVIEW

File: doc/host_create.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ Host Creation
 Host creation is one of the most complex commands in the hammer's foreman plugin. The majority of its parameters
 is obvious but there are some that need further explanation as their values differ with the type of a compute resource.
 
+**Warning:**
+Output of `hammer host create -h` can differ when you're runing your hammer against foreman instances with various versions
+or plugins installed. Hammer adapts according to the api documentation returned from the server. Following is help output
+for `hammer-cli-foreman 0.2.0` run against `foreman 1.8`.
+
```
Comment (mbacovsky) [inline doc/host_create.md]: I'd prefer to re-phrase, but I have no idea which version is more correct ;)
```
Output of `hammer host create -h` may vary with the Foreman server used and plugins installed.

Sample help output for `hammer-cli-foreman 0.2.0` and `Foreman 1.8` follows:
```

**Comment by mbacovsky** [comment]
:+1: with whatever wording you choose

File: doc/host_create.md (hunk 1 above)
Comment (domcleal) [inline doc/host_create.md]: @mbacovsky's sounds clearer to me.

**Comment by tstrachota** [comment]
@mbacovsky updated

**Pull request merged**